lib/errors.js

```diff
@@ -109,7 +108,8 @@ exports.Err = class {
 
         return format.replace(/\{\{(\!?)([^}]+)\}\}/g, ($0, isSecure, name) => {
 
-            const value = Hoek.reach(this.context, name);
+            const context = Object.assign({}, this.context, { key: this.context.label });
```

Comment:
This is precisely the kind of hack I didn't want to do. You're breaking the expectations of the template language by enforcing that `key` is now `label`, by doing that you can't ever use the actual `key`.